tests/validation/tests/v3_api/test_run_sonobuoy.py

```diff
@@ -25,7 +26,10 @@ def test_sonobuoy_results():
 
 
 def run_sonobuoy_test(kubeconfig):
-    cmd = "sonobuoy run --mode={0} --kube-conformance-image-version={1} --kubeconfig={2}".format(RANCHER_SONOBUOY_MODE, RANCHER_K8S_VERSION, kubeconfig)
+    if not RANCHER_FAILED_TEST:
+        cmd = "sonobuoy run --mode={0} --kube-conformance-image-version={1} --kubeconfig={2}".format(RANCHER_SONOBUOY_MODE, RANCHER_K8S_VERSION, kubeconfig)
+    else:
+       cmd = "{0} --kube-conformance-image-version={1} --kubeconfig={2}".format(RANCHER_FAILED_TEST, RANCHER_K8S_VERSION, kubeconfig)
```

Comment:
If we are only ever going to run this as `sonobuoy run --e2e-focus "<failed test>"` then can we change the command to have that part so that we only have to pass in the test name?:
`cmd = 'sonobuoy run --e2e-focus "{0}" --kube-conformance-image-version={1} --kubeconfig={2}'.format(RANCHER_FAILED_TEST, RANCHER_K8S_VERSION, kubeconfig)`